components/cards/StreamingInfo.js

```diff
@@ -0,0 +1,44 @@
+import * as React from "react";
+import Card from "react-bootstrap/Card";
+import Image from "react-bootstrap/Image";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+import { Container } from "react-bootstrap";
+
+function StreamingInfo({ ETHxBalance }) {
+  return (
+    <Card border="secondary" className="bg-dark my-5">
+      <Card.Body>
+        <Container>
+      <Row>
+        <Col xs={2} style={{ alignSelf: 'center' }}>
+          <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-exclamation-circle-fill" viewBox="0 0 16 16">
+            <path d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM8 4a.905.905 0 0 0-.9.995l.35 3.507a.552.552 0 0 0 1.1 0l.35-3.507A.905.905 0 0 0 8 4zm.002 6a1 1 0 1 0 0 2 1 1 0 0 0 0-2z"/>
+          </svg>
+        </Col>
+        <Col>
+          <p>
+            Your current ETHx balance is <b>{ETHxBalance}</b>.
+            We recommend wrapping enough ETH to cover at least&nbsp;
+            3 months of payments for ALL of your outstanding streams&nbsp;
+            plus any required 1-time payments and deposits.
+          </p>
+        </Col>
+      </Row>
+      <Row>
+        <Col xs={2}>{' '}</Col>
+        <Col>
+          <p>
+            <a href={'TODO'} target="_blank" rel="noreferrer">
```

Comment:
Hi @codynhat @gravenp 

What should the **Payments Dashboard** link be here?

Reply:
It will be an internal link to the page created in https://github.com/Geo-Web-Project/cadastre/issues/104.

Thanks for jumping in on these bounties! If you are interested in working on any others, comment on the ticket, so I can assign it and make sure no one else starts working on it.

Reply:
@gravenp If this dashboard is unfinished, then I think the PR is ready for review.

I will comment on the issue if I will work on it.